openjdk-runtime/openjdk8/Dockerfile

```diff
@@ -16,9 +16,9 @@ FROM gcr.io/google_appengine/base
 ENV DEBIAN_FRONTEND noninteractive
 
 # Create env vars to identify image
-ENV GAE_IMAGE_NAME ${project.artifactId}
-ENV GAE_IMAGE_LABEL ${docker.label.prefix}8-jre
-ENV GAE_IMAGE_VERSION ${project.version}
+# ENV GAE_IMAGE_NAME ${project.artifactId}
```

Comment:
Agreed, there should be a standard across images for all supported languages. However, I'm not sure if this would be achieved with environment variables or something else in the platform. In any case, at the moment, these environment variables are not in use.